package.json

```diff
@@ -59,6 +59,7 @@
     "react-dom": "^18.2.0",
     "samlify": "^2.8.8",
     "tiny-invariant": "^1.3.1",
+    "validator": "^13.9.0",
```

Comment:
If we're going to pull in this dependency, we could use it to replace the IPv4 and IPv6 dependency we brought in before.  That is, this module can do a lot of different checks for us, so we can reduce the number of dependencies we have.

Can you file an issue to update that later?

Reply:
Yes, I will